subsys/logging/frontends/Kconfig

```
@@ -74,6 +74,15 @@ config LOG_FRONTEND_STMESP_GUARANTEED_ACCESS
 	  When enabled, accessing STMESP registers will stall if write cannot be
 	  performed (e.g. ETR buffer is full).
 
+config LOG_FRONTEND_STMESP_MSG_END_TIMESTAMP
+	bool "Generate timestamp and marker at message end"
+	help
+	  Generate a zero data byte with timestamp and marker at message end, 
+	  instead of generating a timestamp and marker at message
```

Try to kind of justify the help description

```suggestion
	  instead of generating a timestamp and marker at message start and
```

---

@jilaypandya Thanks, Kconfig entry cleaned up.